components/server/src/redis/mutex.ts

```diff
@@ -20,7 +20,7 @@ export class RedisMutex {
 
             // The max number of times Redlock will attempt to lock a resource
             // before erroring.
-            retryCount: 3,
+            retryCount: 20,
```

Comment:
increasing the retry count to 20, so attempts for getting a mutex will fail after 20 * 200 ms now. This is so that concurrent requests don't immediately error but get blocked at most 4 secs.